src/jepsen/voltdb.clj

```diff
@@ -131,36 +129,39 @@
                                  );"
                       init-schema-file "init-schema"]
                   (cu/write-file! init-schema init-schema-file)
-                  (c/exec (str "JAVA_HOME=/opt/oracle_java17 PATH=/opt/oracle_java17/bin:/usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/sbin:/bin " base-dir "/bin/voltdb")
+                  (c/exec (c/env {"JAVA_HOME" (System/getenv "JAVA_HOME")
+                                  })
+                          "bin/voltdb"
```

Comment:
Are we in base-dir at this point? is "." or "base-dir" in the PATH? otherwise we might not find "./bin/voltdb" from the directory base-dir.

Reply:
we are in jepsen-volt on the machine the test starts in, it's isn't compiled on the worker nodes, only locally